src/Atmos/Model/moisture.jl

```diff
@@ -161,6 +162,18 @@ function gradvariables!(moist::NonEquilMoist, atmos::AtmosModel, transform::Vars
   T = air_temperature(phase)
   e_tot = state.ρe * ρinv
   transform.moisture.h_tot = e_tot + R_m*T
+  transform.moisture.q_liq = state.moisture.ρq_liq * ρinv
+  transform.moisture.q_ice = state.moisture.ρq_ice * ρinv
+end
+
+function flux_moisture!(moist::NonEquilMoist, atmos::AtmosModel, flux::Grad, state::Vars, aux::Vars, t::Real)
+  ρ = state.ρ
+  u = state.ρu / ρ
+  z = altitude(atmos.orientation, aux)
+  usub = subsidence_velocity(atmos.subsidence, z)
+  ẑ = vertical_unit_vector(atmos.orientation, aux)
+  u_tot = u .- usub * ẑ
```

Comment:
Subsidence is being reformulated as a source (in #701) so you may want to mirror `flux_moisture!` for `EquilMoist` in that PR.

Reply:
Sure I'll have this updated once it's working as intended.